src/pages/community/events.js

```diff
@@ -43,8 +43,8 @@ export default () => {
                     {/* TODO: Add FilterBar */}
                     <H3>All Events</H3>
                 </EventsFilter>
-                {/* TODO: remove below when events_api is working */}
-                <EventsList items={[...sampleEvents, ...sampleEvents]} />
+                {/* TODO: update below to not use sample data when events_api is working */}
+                <EventsList items={[...sampleEvents]} />
```

Comment:
Does this code and comment still apply? We're doing all the sample stuff in the hook now

Reply:
Good point, we can render events straight from the hook now.